erica/domain/FreischaltCode/FreischaltCode.py

```diff
@@ -1,27 +1,20 @@
 from abc import ABC
 from datetime import date
 
-from erica.domain.erica_request.erica_request import BasePayload
+from erica.domain.Shared.BaseDomainModel import BasePayload
 
 
 class FreischaltCodeRequestPayload(BasePayload, ABC):
-    tax_ident: str
+    idnr: str
```

Comment:
idnr is the tax identification number or Steuer-IdNummer. That's where the abbreviation comes from. I agree that we should have one consistent naming, that's why I reused the old naming to make it consistent with all our code. However, I can see that `idnr` can be confusing. What about `tax_identification_number` or `tax_id_number`. `tax_ident` confuses me because I am unsure if that means some kind of ident procedure or the the tax identification number.

Reply:
`tax_identification_number` or `tax_id_number` are both good, I think.